charts/newrelic-infrastructure/templates/controlplane/daemonset.yaml

```diff
@@ -93,61 +102,52 @@ spec:
           resources: {{ toYaml . | nindent 12 }}
           {{- end }}
         - name: forwarder
-          image: "{{ .Values.images.forwarder.registry }}/{{ .Values.images.forwarder.repository }}:{{ .Values.images.forwarder.tag }}"
+          image: {{ include "common.images.image" ( dict "imageRoot" .Values.images.forwarder "context" .) }}
           imagePullPolicy: {{ .Values.images.forwarder.pullPolicy }}
           securityContext:
-            {{- mustMergeOverwrite .Values.securityContext  (include "newrelic.compatibility.securityContext" . | fromYaml) | toYaml | nindent 12 }}
+            {{- mustMergeOverwrite
+                    (include "common.securityContext.container" . | fromYaml )
+                    (include "newrelic.compatibility.securityContext" . | fromYaml )
+                    | toYaml | nindent 12 }}
           ports:
-            - containerPort: 8001
+            - containerPort: {{ get (fromYaml (include "nriKubernetes.controlPlane.agentConfig" .)) "http_server_port" }}
           env:
+            - name: "NRIA_LICENSE_KEY"
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "common.license.secretName" . }}
+                  key: {{ include "common.license.secretKeyName" . }}
+
             - name: "NRIA_OVERRIDE_HOSTNAME_SHORT"
               valueFrom:
                 fieldRef:
                   apiVersion: "v1"
                   fieldPath: "spec.nodeName"
-            - name: "NRIA_HTTP_SERVER_PORT"
-              value: "8001"
+
             - name: "K8S_NODE_NAME"
               valueFrom:
                 fieldRef:
                   apiVersion: "v1"
                   fieldPath: "spec.nodeName"
-            {{- if .Values.verboseLog }}
-            - name: NRIA_VERBOSE
-              value: "1"
-            {{- end }}
+
             {{- if .Values.useNodeNameAsDisplayName }}
             - name: "NRIA_DISPLAY_NAME"
```

Comment:
shouldn't this config also moved to the helper?

Reply:
You are right. But I still have to change this from this common library to 1.0 and didn't want to spend more time on this and make this PR bigger.

I leave this comment open as a reminder of doing it in the follow-up PR.